elements/pfe-modal/src/pfe-modal.js

```diff
@@ -164,7 +164,9 @@ class PfeModal extends PFElement {
     this._overlay.removeAttribute("hidden");
     this._outer.removeAttribute("hidden");
     // This prevents background scroll
-    document.body.style.overflow = "hidden";
+    document.body.style.overflow = "scroll";
```

Comment:
These changes on `document.body` scroll the user back to the top of the page when the modal is opened. Is this change for the jumpiness when opening a modal?